app/static/eventdisplay.js

```diff
@@ -0,0 +1,24 @@
+function scrollTo(element) {
+    window.scroll({
+      behavior: 'smooth',
+      left: 0,
+      top: element.offsetTop
+    });
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+    let chosen_button = document.getElementsByClassName('event-btn');
```

Comment:
Can be `const`

Reply:
fixed all of the notes:, waiting for merge :)